frontend/src/components/elements/VegaLiteChart/VegaLiteChart.tsx

```diff
@@ -48,6 +49,7 @@ const SUPPORTED_INDEX_TYPES = new Set([
 
 interface Props {
   width: number
+  height: number | undefined
```

Comment:
- I'm not seeing the expand button on the first 4 vega lite charts in `e2e/scripts/vega_lite_chart.py`, only the last 4, is this expected?
- The charts expand to fullscreen height but not width, is this expected?
- If you expand a chart, then set the width to 500, then re-run report, the chart will expand to the width but revert it's height. If you set the width before expanding the chart, it expands correctly. Very small use case, maybe not worth the hassle.

Reply:
> I'm not seeing the expand button on the first 4 vega lite charts
You are right, there are a bug with the opacity transition, I will try to solve it.

Reply:
> The charts expand to fullscreen height but not width, is this expected?

That is right, because the user are specifying a width into the script

Reply:
> If you expand a chart, then set the width to 500, then re-run report, the chart will expand to the width but revert it's height. If you set the width before expanding the chart, it expands correctly. Very minor case.

I will try to solve it